test/CMakeLists.txt

```diff
@@ -173,6 +173,7 @@ function(add_gtest_executable TEST_NAME)
 endfunction()
 
 add_compile_options(-Wno-c++20-extensions)
+add_subdirectory(tile)
```

Comment:
```suggestion
add_subdirectory(ck_tile)
```

Reply:
I think tile is more better since directory for ck_wrapper has name wrapper. Is it ok?